src/OpenTelemetry/Trace/TracerProviderSdk.cs

```diff
@@ -33,34 +33,29 @@ internal class TracerProviderSdk : TracerProvider
         private readonly List<object> instrumentations = new List<object>();
         private readonly ActivityListener listener;
         private readonly Sampler sampler;
-        private readonly ActivitySourceAdapter adapter;
+        private readonly HashSet<string> legacyActivityOperationNames;
         private BaseProcessor<Activity> processor;
+        private Action<Activity> getRequestedDataAction;
+        private bool needInstrumentations;
```

Comment:
```suggestion
        private bool supportLegacyActivity;
```
```suggestion
        private bool needInstrumentations;
```

Reply:
Renamed the field to supportLegacyActivity.